src/widgets/communication/ui/communication.tsx

```diff
@@ -47,26 +54,36 @@ export const Communication = ({ courseName, lang = 'en' }: CommunicationProps) =
             <Image src={discordLogo} alt="discord logo" />
           </figure>
           <div>
-            <Subtitle>{subTitle}</Subtitle>
-            <Paragraph className={cx('communication-paragraph')}>
-              {firstParagraphFirstHalf}
-              <LinkCustom href={DISCORD_LINKS[courseName]} external>
-                {discordLink}
+            <Subtitle className={cx('communication-subtitle')}>
+              {isJsEnCourse ? subTitleJs : subTitle}
```

Comment:
please, move all the conditions above the `return` statement
```tsx
const courseSubTitle = isJsEnCourse ? subTitleJs : subTitle;
```

```suggestion
              {subTitle}
```

Reply:
Done